### PR TITLE
chore: update bridge event status endpoint

### PR DESCRIPTION
**Pull request by flagrede**
## Description

Closes: regen-network/regen-web#1755

- update status endpoint from: `${apiUri}/regen/events/${hash}/status` to `${apiUri}/regen/events/${hash}`
- update fields in response: 
  - `tx_hash`-> `regen_tx_hash`
  - `destination_tx_hash` -> `evm_tx_hash`

---

### Author Checklist

_All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues._

I have...

- [ ] provided a link to the relevant issue or specification
- [ ] provided instructions on how to test
- [ ] reviewed "Files changed" and left comments if necessary
- [ ] confirmed all CI checks have passed
- [ ] once the PR is closed, set up backport PRs for `redwood`, `hambach` and optionally `master` (see below)

#### Setting up backport PRs

After merging your PR to `dev`, set up backports by doing the following:

1. If your branch does not have merge commits, add the following comment to
   your PR:

- If PR is a new feature: `@Mergifyio backport redwood hambach`.
- If PR is a bug fix: `@Mergifyio backport redwood hambach master`.

2. If your branch does have merge commits:

   a. Pull latest `dev`, `hambach` and `redwood` branches

   b. Create new branches for backports and merge `dev` (replace `<PR#>` with your PR #)

   ```
   git checkout -b hambach-backport-<PR#> hambach
   git merge dev
   git push origin hambach-backport-<PR#>
   git checkout -b redwood-backport-<PR#> redwood
   git merge dev
   git push origin redwood-backport-<PR#>`
   ```

   c. Open new PRs in regen-web targeting `hambach` and `redwood`, respectively.

   d. If PR is a bug fix repeat the same process for `master` branch.

### How to test

1.

### Reviewers Checklist

_All items are required. Please add a note if the item is not applicable and please **add
your handle next to the items reviewed if you only reviewed selected items**._

I have...

- [ ] confirmed all author checklist items have been addressed
- [ ] reviewed code correctness and readability
- [ ] verified React components follow DRY principles
- [ ] reviewed documentation is accurate
- [ ] reviewed tests
- [ ] manually tested (if applicable)

### CODE REVIEW

File: web-registry/src/lib/bridge.ts [hunk 1]
```diff
@@ -6,8 +6,8 @@ export const getBridgeTxStatus = async (
   hash: string,
 ): Promise<
   | {
-      tx_hash: string;
-      destination_tx_hash: string;
+      regen_tx_hash: string;
+      emv_tx_hash: string;
```
Comment (blushi) [inline web-registry/src/lib/bridge.ts:10]:
```suggestion
      evm_tx_hash: string;
```

File: web-registry/src/lib/ecocredit/api.ts [hunk 2]
```diff
@@ -240,7 +240,7 @@ export const getBridgedEcocreditsForAccount = async (
         const txStatus = await getBridgeTxStatus(res.txResponses[i].txhash);
 
         status = txStatus?.status;
-        destinationTxHash = txStatus?.destination_tx_hash;
+        destinationTxHash = txStatus?.emv_tx_hash;
```
Comment (blushi) [inline web-registry/src/lib/ecocredit/api.ts:243]: what about also renaming the variable ?
```suggestion
        evmTxHash = txStatus?.evm_tx_hash;
```

**Review by blushi — CHANGES_REQUESTED** [review]

File: web-registry/src/lib/bridge.ts (hunk 1 above)
Comment (flagrede) [reply web-registry/src/lib/bridge.ts:10]: Nice catch.

File: web-registry/src/lib/ecocredit/api.ts (hunk 2 above)
Comment (flagrede) [reply web-registry/src/lib/ecocredit/api.ts:243]: I thought it was fine here to keep `destination` here as on the front end we do not necessarily need to know that it is an EVM tx hash.

Comment (blushi) [reply web-registry/src/lib/ecocredit/api.ts:243]: ok for me too but let's fix the typo

**Review by blushi — APPROVED** [review]
tACK

**Comment by flagrede** [comment]
@Mergifyio backport redwood hambach

**Pull request merged**